.gitignore

```diff
@@ -1,24 +1,36 @@
 # dependencies
 /node_modules
+server/node_modules
 /.pnp
 .pnp.js
 
+# config
+server/config.env
+
+.npm
+
 # testing
 /coverage
 
 # production
 /build
 
 # misc
-.DS_Store
+*/.DS_Store
```

Comment:
undo change please, i think `.DS_Store` is correct

Reply:
oh my ide was trying to push ds_store in some sub_dir. **/.DS_Store should gitignore all of them